Sources/Orbit/Components/InputField.swift

```diff
@@ -173,6 +197,90 @@ public extension InputField {
         onEditingChanged: @escaping (Bool) -> Void = { _ in },
         onCommit: @escaping () -> Void = {},
         suffixAction: (() -> Void)? = nil
+    ) {
+        self.init(
+            label,
+            value: value,
+            prefix: prefix,
+            suffix: suffix,
+            placeholder: placeholder,
+            state: state,
+            textContent: textContent,
+            keyboard: keyboard,
+            autocapitalization: autocapitalization,
+            isAutocompleteEnabled: isAutocompleteEnabled,
+            passwordStrength: passwordStrength,
+            message: message,
+            messageHeight: messageHeight,
+            mode: .actionsHandler(onEditingChanged: onEditingChanged, onCommit: onCommit, isSecure: isSecure),
+            suffixAction: suffixAction
+        )
+    }
+
+    /// Creates Orbit InputField component.
+    ///
+    /// - Parameters:
+    ///     - message: Message below InputField.
+    ///     - messageHeight: Binding to the current height of message.
+    ///     - formatter: A formatter to use when converting between the
+    ///     string the user edits and the underlying String value.
+    ///     If `formatter` can't perform the conversion, the text field doesn't
+    ///     modify `binding.value`.
+    ///     - suffixAction: Optional separate action on suffix icon tap.
+    init(
+        _ label: String = "",
+        value: Binding<String>,
+        prefix: Icon.Content = .none,
+        suffix: Icon.Content = .none,
+        placeholder: String = "",
+        state: InputState = .default,
+        textContent: UITextContentType? = nil,
+        keyboard: UIKeyboardType = .default,
+        autocapitalization: UITextAutocapitalizationType = .none,
+        isAutocompleteEnabled: Bool = false,
+        passwordStrength: PasswordStrengthIndicator.PasswordStrength = .empty,
```

Comment:
The formatter variant probably does not need passwordStrength as it does not support secure field